lib/util/logger.js

```diff
@@ -70,19 +70,21 @@ function ConsoleColor() {
 
   Object.keys(this.foreground_colors).forEach(function(k) {
     ConsoleColor.prototype[k.toLowerCase()] = function Foreground(text, background) {
-      var string = '\033[' + self.foreground_colors[k.toLowerCase()] + 'm';
+      // Avoid Parsing error: Octal literal in strict mode by using string concatenation
+      var string = '\0' + '33[' + self.foreground_colors[k.toLowerCase()] + 'm';
```

Comment:
This trick is not very satisfying. You could use the hex value instead:
`var string = '\x1B[' + self.foreground_colors[k.toLowerCase()] + 'm';`